### PR TITLE
Collect dmesg changes and print them at the end as well.

### PR DESCRIPTION
**Pull request by vext01**
Not paying much attention to which fd we print errors on at this point, as I will be changing everything to use the logging API next.

Cheers

### CODE REVIEW

**Comment by vext01** [comment]
Example output: https://gist.github.com/vext01/1dc50fcfd41ceb1882e5

File: krun/platform.py [hunk 1]
```diff
@@ -21,21 +21,49 @@ def __init__(self):
         # errors and warnings in the Linux dmesg. If that happens, we
         # really want to know about it!
         self.current_dmesg = self._collect_dmesg_lines()
+        self.current_dmesg_time = time.time()
```
Comment (ltratt) [inline krun/platform.py]: Why use time() here but localtime() later.

Comment (ltratt) [inline krun/platform.py]: I'm also not sure `current_dmesg_time` is a good name: shouldn't it be something like `last_dmesg_time`?

**Comment by ltratt** [comment]
The commit title is wrong ("aswell" instead of "as well") -- a force push of the commit should fix this.

**Comment by ltratt** [comment]
Are all the dmesg changes printed at the end of a run too? [If things scroll off the top of the screen, people won't notice. I also wonder if problems like this should be emailed to a specified address(es) so that people don't need to continually keep an eye on the console?]

File: krun/platform.py (hunk 1 above)
Comment (vext01) [inline krun/platform.py]: former: I'm using it to convert to local time, but I see a workaround. (time api in python is awful).

latter: as you wish

**Comment by vext01** [comment]
aswell -> as well: this really doesn't matter. If you absolutely insist, one we agree the changes are ok,  I will merge by and and amend the commit. We should never force push.

**Comment by vext01** [comment]
dmesgs are printed at the end. yes.

**Pull request merged**